cypress/integration/settings.js

```diff
@@ -0,0 +1,12 @@
+it('should be able to open and close the settings modal', () => {
+    cy.visit('/')
+        .get('[data-testid=component-name]')
+        .get('#settingsPanelOpenButton')
```

Comment:
Might be cleaner to wait for the setting button to appear
```suggestion
        .get('#settingsPanelOpenButton')
        .should('be.visible')
```